userbenchmark/ddp_experiments/__init__.py

```diff
@@ -679,7 +685,7 @@ def main():
         slurm_exclude=args.exclude,
     )
 
-    executor.update_parameters(name="distbench", slurm_array_parallelism=1, timeout_min=1000)
+    executor.update_parameters(name="distbench", slurm_array_parallelism=1, timeout_min=args.timeout)
```

Comment:
cc @ShiboXing I think the timeout issue was actually slightly different. The other timeouts are for things like file-based synchronization, and waiting for initialization. This timeout is for how long before we kill the slurm job. Let me know if this is the change you wanted!

Reply:
got it, this should fix it, thanks!